src/schema/search.ts

```diff
@@ -483,6 +519,9 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
                 })
                 .andWhere('cp."userId" = :cpUserId', {
                   cpUserId: ctx.userId,
+                })
+                .andWhere('cp."feedId" = :cpFeedId', {
```

Comment:
Not 100%, but can you check that user can't just send someone else feedId and search through it, I think above `userId` condition should make it impossible, but just to be sure. Adding a test for that would be golden!

Reply:
Same for source search

Reply:
Ah good point, let me add it and validate :)